tools/nnyeah/integration/Makefile

```diff
@@ -0,0 +1,27 @@
+TOP=../../..
+include $(TOP)/Make.config
+
+all-local:: mac/bin/Debug/MacIntegration.dll ios/bin/Debug/iOSIntegration.dll tv/bin/Debug/TVIntegration.dll
```

Comment:
It would be nice to use a consistent platform naming: `macOS`, `iOS`, `tvOS`.

That way it's much easier to handle multiple platforms (we have a list of platforms we're building for with that naming), to:

* Not test platforms we're not building for.
* Maybe one day iterate over all the platforms (as opposed to hardcoding them).

Reply:
Good point. Let me fix that.